src/analysis/retail/priest/holy/constants.ts

```diff
@@ -103,6 +103,16 @@ export const ABILITIES_AFFECTED_BY_APOTHEOSIS_TALENT = [
   TALENTS.HOLY_WORD_CHASTISE_TALENT.id,
 ];
 
+export const ABILITIES_THAT_DONT_TRIGGER_MASTERY = [
+  TALENTS.RENEW_TALENT.id,
+  SPELLS.PREMONITION_OF_PIETY.id,
+  SPELLS.DIVINE_IMAGE_BLESSED_LIGHT_HEAL.id,
+  SPELLS.DIVINE_IMAGE_DAZZLING_LIGHT_HEAL.id,
+  SPELLS.DIVINE_IMAGE_HEALING_LIGHT_HEAL.id,
+  SPELLS.DIVINE_IMAGE_TRANQUIL_LIGHT_HEAL.id,
+  TALENTS.ESSENCE_DEVOURER_TALENT.id,
+];
```

Comment:
rather than listing this out, it might be better to keep the whitelist so that you don't accidentally attribute mastery healing to non-spec heal sources like embellishments, trinkets, or potions.

Reply:
Embellishments/trinkets/potions/etc do contribute to EoL. It is triggered from pretty much anything that isnt a HoT or from a pet, regardless of if it is class specific.

Reply:
oh, interesting. i did not know that. that makes your other use of a null/blacklist rather than whitelist make a ton of sense